CONTRIBUTING.md

```diff
@@ -16,6 +16,45 @@ As described in the [spec][spec] the build will automatically update the `dist`
 $ npm run build
 ```
 
+## Extract Method
```

Comment:
Is it true that we now have two redundant documentations (README.md and CONTRIBUTING.md)?